azure-iot-common/tests/conftest.py

```diff
@@ -0,0 +1,8 @@
+import sys
+
+collect_ignore = []
+
+# Ignore Async tests if below Python 3.5
+if sys.version_info < (3, 5):
+    collect_ignore.append("test_async_adapter.py")
```

Comment:
>test_async_adapter.py [](start = 27, length = 21)

nit/for later: can we use a wildcard and a naming pattern for tests?

Reply:
That would be bad here, given that we want to specifically forbid these files, not collect them. Pytest handles collections automatically.